_assets/gulp_config/paths.js

```diff
@@ -68,4 +68,22 @@ paths.jekyllImageFilesGlob = paths.jekyllImageFiles + paths.imagePattern;
 // Site files globs
 paths.siteHtmlFilesGlob = paths.siteDir + paths.htmlPattern;
 
+// HTML pages to run through the accessibility test.
+paths.htmlTestFiles = [
+  '_site/index.html',
+  '_site/2017/**/*.html',
+  '_site/blog/index.html',
+  '_site/blog/tag/drupal/index.html',
+  '_site/company/index.html',
+  '_site/company/anne-tomasevich/index.html',
+  '_site/company/mission-and-values/index.html',
+  '_site/contact/index.html',
+  '_site/results/index.html',
+  '_site/results/case-studies/index.html',
+  '_site/results/case-studies/duke-today/index.html',
+  '_site/results/open-source/index.html',
+  '_site/services/index.html',
+  '_site/services/development/index.html'
+];
+
```

Comment:
Hmmm, might there be a way to test entire directories or some sort of wildcard. I worry we'll add pages in the future and nothing will prompt us to know to add them to this list.

Also, this is a bit generic, and perhaps related to many different kinds of Travis tasks, but it would be nice to not retest each file each time if they're passed previously. I think something like this would be nice for image generation too - hence the point this is probably broadly applicable. Perhaps it's not suitable to give too much thought to this task, but we should consider if there are ways (like checking a file hash) to see if a given file, image etc ought to be retested, regenerated.